### PR TITLE
[Rails4] fix constant reloading issues: remove unloadable

### PR DESCRIPTION
**Pull request by myabc**
:warning: ~~**Includes commits from #3194. Please review, merge first.**~~

:information_source: **Please see finnlabs/openproject-backlogs#145 for explanation.**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://myabc-feature-rails4-remove-unloadable.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://myabc-feature-rails4-remove-unloadable.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by ulferts** [comment]
@myabc now that #3194 is merged, can you shorten the commit list in this PR, please?

File: lib/open_project/journal_formatter/attachment.rb [hunk 1]
```diff
@@ -27,8 +27,6 @@
 #++
 
 class OpenProject::JournalFormatter::Attachment < ::JournalFormatter::Base
-  # unloadable
```
Comment (ulferts) [inline lib/open_project/journal_formatter/attachment.rb:30]: There are quite a few of those in here @myabc. Can you please get rid of them for good?

Comment (ulferts) [inline lib/open_project/journal_formatter/attachment.rb:30]: Sorry, my fault, I misinterpreted the diff.

Comment (myabc) [inline lib/open_project/journal_formatter/attachment.rb:30]: @ulferts ok! :smile:

**Pull request merged**